Show package version in server UI sidebar

gravi2 opened the pull request:
@patrickhulce I am raising this PR to address #145. 

Below is a quick overview of what has changed:
1.  I am making use of process.env.npm_package_version to make this change at the build time. I assume the `lerna version` lifecycle scripts will update the package.json  and the same will be picked by `parcel` in the `build` command.
2. JSX Changes in page-sidebar.jsx to add the siderbar footer and version elements. 
3. CSS changes in app.css and page-sidebar.css

Attaching few screenshots for your reference.

![desktop](https://user-images.githubusercontent.com/1253458/70605797-03ca3900-1bb0-11ea-88fc-4fa05a2c9141.png)

![mobile](https://user-images.githubusercontent.com/1253458/70605808-088eed00-1bb0-11ea-9ba4-7752a809f174.png)

patrickhulce commented on packages/server/src/ui/app.css:
```diff
@@ -97,6 +97,8 @@
 
   --page-sidebar-width: 260px;
   --page-sidebar-nav-item-height: 24px;
+  --page-sidebar-footer-font-size: 14px;
```
let's not use a different font-size for this, how about `--subtext-font-size`?

gravi2 replied:
@patrickhulce I have accepted your CSS changes and I have removed --page-sidebar-footer-font-size as we don't need it any more.

patrickhulce commented on packages/server/src/ui/app.css:
```diff
@@ -97,6 +97,8 @@
   --page-sidebar-width: 260px;
   --page-sidebar-nav-item-height: 24px;
+  --page-sidebar-footer-font-size: 14px;
+  --page-sidebar-footer-margin: 25px;
```
we try not to use bespoke spacing, we have `var(--base-spacing)` for margin and padding :)

gravi2 replied:
@patrickhulce I have accepted your CSS changes and I have removed --page-sidebar-footer-maring as we don't need it any more.

patrickhulce commented on packages/server/src/ui/layout/page-sidebar.jsx:
```diff
@@ -51,6 +51,9 @@ export const PageSidebar = props => {
         />
       </div>
+      <div className="page-sidebar__footer">
+        <div className="page-sidebar__version">Version {process.env.npm_package_version}</div>
```
```suggestion
        <div className="page-sidebar__version">
          {/* This environment variable is set by npm/yarn on any script command. */}
          v{process.env.npm_package_version}
         </div>
```

patrickhulce commented on packages/server/src/ui/layout/page-sidebar.css:
```diff
@@ -69,3 +69,17 @@
+}
+
+.page-sidebar__version {
+  color:sandybrown;
```
```suggestion
  color: var(--inverted-secondary-text-color);
```

patrickhulce commented on packages/server/src/ui/layout/page-sidebar.css:
```diff
@@ -69,3 +69,17 @@
   text-decoration: none;
 }
+
+.page-sidebar__footer {
```
```suggestion
.page-sidebar__footer {
  position: absolute;
  left: 0;
  right: 0;
  bottom: var(--base-spacing);
  text-align: center;
}
```

maybe something like that?

gravi2 commented:
@patrickhulce Thanks for your review comments. I have addressed the review feedback and the same is now available in the PR. Here is the latest screenshot after review changes, for your reference. 
![Screenshot from 2019-12-12 01-22-30](https://user-images.githubusercontent.com/1253458/70699592-3b9fb200-1c7e-11ea-8c28-2d69e8664d52.png)

gravi2 commented:
@patrickhulce I have fixed the linting issue using the autofix (--fix). Though I din't quite like the lint fix it did :)

patrickhulce commented:
>  Though I din't quite like the lint fix it did :)

Haha yeah that one doesn't make the most sense to me either, looks like there's an open bug in Prettier for it already. (https://github.com/prettier/prettier/issues/7132#issuecomment-565489232)


Thanks @gravi2!
